content/en/docs/cluster-administration/Shuting Down and Restart Cluster cracefully.md

```diff
@@ -42,32 +41,34 @@ done
 ```
 Then you can shut down other cluster dependencies, such as external storage.
 
-## Restart the Cluster Gracefully
-This is about restarting the cluster gracefully after shutting down cluster gracefully.
+## Restart Cluster Gracefully
+You can restart a cluster gracefully after shutting down the cluster gracefully.
```

Comment:
@ruiyaoOps  @Sherlock113 @pixiake   Never hear about the word `restart a cluster`, is it more exact to update it to `restart all nodes in the cluster`?

Reply:
![image](https://user-images.githubusercontent.com/65327072/96941575-592be200-1505-11eb-921d-bd728e5dfecd.png)
Please have a look at this, the [OpenShift doc](https://docs.openshift.com/container-platform/4.5/backup_and_restore/graceful-cluster-restart.html).